t/t1025-flux-account-projects.t

```diff
@@ -43,6 +43,16 @@ test_expect_success 'add some queues to the DB' '
 	flux account add-queue special --priority=99999
 '
 
+test_expect_success 'list contents of project_table before adding projects' '
+	flux account list-projects > project_table.test &&
+	cat <<-EOF >project_table.expected
+	project_id | project | usage
+	-----------+---------+------
+	1          | *       | 0.0
```

Comment:
Usage is reported in what unit?

Reply:
Usage in accounting is calculated as a product of `nnodes` and `t_elapsed` (which is defined as `t_inactive - t_run`). I have some more detailed explanations and examples [here](https://github.com/flux-framework/flux-accounting/tree/master/src/bindings/python/fluxacct/accounting).